sky/services/editing/editing.mojom

```diff
@@ -48,3 +48,13 @@ interface Keyboard {
   Show();
   Hide();
 };
+
+struct ClipData {
+  string text;
+};
+
+[ServiceName="editing::Clipboard"]
+interface Clipboard {
+  SetClipData(ClipData clip);
+  GetClipData() => (ClipData? clip);
```

Comment:
Yeah, there might be a large amount of data on the clipboard (e.g., an image or a video) that's not worth transferring if you really wanted some text.  Another use case is if there are multiple representations of the object on the clipboard (e.g., rich text or plain text), you only actually want to generate the representation that the client can understand.